editor/src/core/workers/parser-printer/parser-printer.test-utils.ts

```diff
@@ -881,56 +887,131 @@ function walkElementsWithin(
   elementsWithin: ElementsWithin,
   includeDataUIDAttribute: IncludeDataUIDAttribute,
   walkWith: (elem: JSXElementChild) => void,
+  shouldWalkElement: (elem: JSXElementChild) => boolean,
+  shouldWalkAttributes: boolean,
```

Comment:
This should probably be a union type like `'walk-attributes' | 'do-not-walk-attributes'`.

Reply:
Good shout! Done 👍